var/spack/repos/builtin/packages/ncurses/package.py

```diff
@@ -87,12 +87,6 @@ def build(self, spec, prefix):
         with working_dir('build_ncursesw'):
             make()
 
-    def check(self):
-        with working_dir('build_ncurses'):
-            make('check')
-        with working_dir('build_ncursesw'):
-            make('check')
```

Comment:
The `ncurses` Makefile doesn't have a `check` target. This was causing `spack install --run-tests ncurses` to crash.